src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -72,6 +72,39 @@ public Type[] GetStartupTypes()
 
         internal bool HasExternalConfigurationStartups() => _startupTypes.Value.Any(p => typeof(IWebJobsConfigurationStartup).IsAssignableFrom(p));
 
+        public static string FindExtensionsMetadataPath(string rootScriptPath, out string baseProbingPath)
+        {
+            baseProbingPath = null;
```

Comment:
Just refactoring this so it can be -reused as per: https://github.com/Azure/azure-functions-host/pull/9331#discussion_r1232543085